### DISCUSSION TITLE
About the issue of name release after account cancellation

### DISCUSSION THREAD

**Discussion by vchenpeng** [npm/feedback#714] [Registry] [answered]
After actively deleting the account, how long does it take for the user name to be released and allowed to be registered again?

I deleted my account, but I regret it and want to register again

**Comment by MylesBorins** [comment, marked as answer]
[Please open a ticket with support](https://www.npmjs.com/support) and they can potentially restore your account. We cannot guarantee the account can be recovered otherwise.
